### ISSUE TITLE
E126: behaves differently when part of if statement

### ISSUE THREAD

**Issue by PeterJCLaw** [PyCQA/pycodestyle#943]
Given the considerable previous issues around `E126`, I'm not sure if this is a bug or just my expectation being wrong.

`E126` seems to behave differently for this:
``` python
wiblle(
        'arg',
        'arg2',
)
```
which produces an error, as I would expect, than for this:
``` python
if not wiblle(
        'arg',
        'arg2',
):
    pass
```
which doesn't produce an error.

In both cases, the arguments to `wibble` are indented by eight spaces.

It's not that `E126` is silent in the if statement case -- changing the indent there to any value outside `(4, 8)` seems to trigger it.

I'm _guessing_ that this might be here to allow this example from PEP 8:
``` python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

However I would contend that this wrapping case is sufficiently different that the extra indentation should never be needed and thus rejected.
The key differences are:
- the parens are on their own lines than the tokens between them, and
- the parens are part of a function call

(aside: IMO this style is clear on its own and overall clearer anyway)

Is this behaviour with `if`s expected and/or would you be open to emitting `E126` here?

**Comment by hoylemd** [comment]
I'm personally a big fan of the
```
wiblle(
    arg1,
    arg2
)
```
pattern because it makes it really easy to manipulate the arguments.

I wouldn't be surprised if this gets rejected due to not being explicitly stated in pep8 though.  I think the design goals of pycodestyle are more of a strict implementation of pep8.  A flake8 plugin might be a more appropriate medium

**Comment by asottile** [comment]
at least for if statements, double-indenting is allowed as it differentiates from the code block underneath it
